content/en/about/overview.md

```diff
@@ -10,45 +10,71 @@ weight: 1
 
-It’s free to use for all developers and software providers, with Sigstore’s code and operational tooling being 100% open source, and everything maintained and developed by the Sigstore community.
+The project is backed by the Open Source Security Foundation (OpenSSF) under the Linux Foundation, with contributions from Google, Red Hat, Chainguard, GitHub and Purdue University. It is 100% open source and free to use for all developers and software providers. The Sigstore community develops and maintains tools to simplify code signing and verification, and also operates a public-good, non-profit service to improve the open source software supply chain.
 
-## How to use Sigstore
+## Why cryptographic signing?
+
+In a landscape of growing software supply chain vulnerability, unsigned software is at risk for several attack vectors, such as:
+
+- **Typosquatting**
+- **Packages with similar names**
+- **Compromised site where package is hosted**
+- **Tampering after being published**
+
+Digital signatures are a way to verify the authenticity of a software artifact. Software consumers can trace software back to the source to know who created the artifact and that it has not been altered or tampered with after it was signed.
+
+## Why Sigstore?
+
+Traditional artifact signing relies on exchanging cryptographic keypairs for signature verification. The software creator keeps one key secret (the private “signing” key) and publishes the other (the public “verification” key). When a software consumer wants to verify an artifact’s signature, the verification keys are exchanged to prove that the holder of the private key created the signature.
+
+This traditional approach has several weaknesses:
 
-* I want to [Quick Start](/signing/quickstart/)
-* I want to [sign a blob](signing/signing_with_blobs/)
-* I want to [sign a container](signing/signing_with_containers/)
-* I want to [Sign Git commits with Gitsign](/signing/gitsign/)
-* I want to [verify entries with Cosign](/verifying/verify/)
+- **Identity**: How do you know the person signing the artifact is who they say they are?
+- **Key management**: How do you keep the private key secure so it can’t be lost or stolen? How do you make the public key easily accessible for users, but also protect it from tampering by a malicious attacker?
+- **Key revocation**: If the keypair is compromised, how do you distribute new keys in a way that convinces users of your legitimacy and that you’re not an attacker?
+
+Sigstore addresses these problems by helping users move away from a key-based signing approach to an identity-based one. When using Sigstore’s full capabilities, your artifact is:
+
+- **Signed**: By using a Sigstore client (Cosign).
+- **Verified**: By checking your identity with our certificate authority (Fulcio).
```

Comment:
I want to be careful to say that the artifact is verified, because it's not. Sigstore doesn't assert anything about the artifact's legitimacy. For example, someone may sign malware or a tampered artifact with a valid identity.

I'm also hesitant to say that the identity is verified by Fulcio. The CA only issues certificates to an identity token that is valid, but it doesn't assert that the ID token was issued in a valid way (for example, if I can compromise an account, I can request a token still).

Maybe instead, we say something about associating an identity with an ephemeral key? How do we simplify that into one word?

cc @olivekl

Reply:
> Maybe instead, we say something about associating an identity with an ephemeral key? How do we simplify that into one word?
> 
> cc @olivekl

"Coordinate the identity with the associated ephemeral key"   ?   Maybe.   I don't think one word covers this concept.

Reply:
I replaced the "verified" sentence with:

**Associated**: The identity is associated with an ephemeral key.

Not sure what else to say right now.

Reply:
I like that!

Reply:
Sorry, late to the party here, but just some quick thoughts: would "verifiable" make more sense? I'm not sure "Associated" means anything out of context, whereas "Verifiable" or "Able to be verified" gives a sense of what "associated with an ephemeral key" buys you.

I think "associated with an ephemeral key" almost makes it sound like that quality is, well, ephemeral and will disappear.